### PR TITLE
Shadowbringers 5.0 new skills

### PR DESCRIPTION
**Pull request by ShammyLevva**
Added new skills from Shadowbringers.

Note a couple of the icons are missing as they don't seem to be on Garland Tools yet. Namely preparatoryTouch, delicateSynthesis, intensiveSynthesis.

Reuse, Trained Eye & Trained Instinct have specific new functions that will need dealt with but deal with getting materials back or recipes 10 levels below current crafter level.

Rapid Synthesis 3 has a caveat that "efficiency decreases significantly when durability is less than 20". This is a 20 durability cost ability but grants 600% efficiency to progress. Wording suggests if used when durability is 5, 10 or 15 then the efficiency will be less. As yet untested as to how much less.

### CODE REVIEW

File: app/js/services/actions.js [hunk 1]
```diff
@@ -80,6 +81,15 @@
...
+    reuse: {                   common: true,             skillID: {"Alchemist": 4603, "Armorer": 4599, "Blacksmith": 4598, "Carpenter": 4597, "Culinarian": 4604, "Goldsmith": 4600, "Leatherworker": 4602, "Weaver": 4601} },
+    delicateSynthesis: {       common: true,             skillID: {"Alchemist": 100329, "Armorer": 100325, "Blacksmith": 100324, "Carpenter": 100323, "Culinarian": 100330, "Goldsmith": 100326, "Leatherworker": 100327, "Weaver": 100328} },
+    intensiveSynthesis: {      common: true,             skillID: {"Alchemist": 100321, "Armorer": 100317, "Blacksmith": 100316, "Carpenter": 100315, "Culinarian": 100322, "Goldsmith": 100318, "Leatherworker": 100319, "Weaver": 100320} },
+    trainedEye: {              common: true,             skillID: {"Alchemist": 100289, "Armorer": 100285, "Blacksmith": 100284, "Carpenter": 100283, "Culinarian": 100290, "Goldsmith": 100286, "Leatherworker": 100287, "Weaver": 100288} },
+    trainedInstinct: {         common: true,             skillID: {"Alchemist": 100297, "Armorer": 100293, "Blacksmith": 100292, "Carpenter": 100291, "Culinarian": 100298, "Goldsmith": 100294, "Leatherworker": 100295, "Weaver": 100296} },
```
Comment (doxxx) [inline app/js/services/actions.js]: Making `reuse`, etc. available for the user add to sequences, but without defining their model attributes, may cause trouble. Is there any point to allowing users to include them in sequences? If not, rather just leave them commented out here as well until they can be properly modeled.

**Comment by doxxx** [comment]
Thank you for submitting this!

File: app/js/services/actions.js (hunk 1 above)
Comment (ShammyLevva) [reply app/js/services/actions.js]: done.

**Comment by doxxx** [comment]
Affects #272.

**Pull request merged**